clients/ballerina/github.com/redis/ballerinax-redis.json

```
@@ -0,0 +1,7 @@
+{
+  "name": "ballerinax-redis",
+  "description": "Official Redis client for Ballerina language with the support for Redis clusters, connection pooling and secure connections.",
+  "homepage": "https://central.ballerina.io/ballerinax/redis/latest",
+  "repository": "https://github.com/ballerina-platform/module-ballerinax-redis",
+  "official": true
```

Official are only the few clients maintained by the Redis community. You can have a heart instead of a star.

```suggestion
  "recommended": true
```

---

I thought that we can mark a client as official, if the client is maintained by the official development team of the related language/framework/platform :) Sure will change accordingly.